contents/blog/introducing-hogql.md

```diff
@@ -0,0 +1,112 @@
+---
+date: 2023-05-08
+title: "Introducing SQL and HogQL access for PostHog"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: PostHog news
+author:
+  - joe-martin
+tags:
+  - Product updates
+  - Release notes
+featuredImage: ../images/blog/hog_ql.png
+featuredImageType: full
+---
+
+Today, we're excited to announce a major new feature which we're making available to all users as a public beta on PostHog Cloud - the ability to directly access data stored in PostHog via SQL.
+
+We've added several new ways to use SQL in PostHog - you can use expressions to enhance insights, you can use SQL to filter a revamped events list, and you can even use full SQL queries to analyse data in any way you want. 
+
+We've even created [our own SQL dialect, called HogQL](/docs/product-analytics/hogql), which enables direct access to data without the insecurities that come from using ClickHouse SQL directly.
```

Comment:
"without the insecurities" is irrelevant from an user's standpoint.

We built HogQL prevent users from accessing one another's data, as everything is in just one database for us. But the users don't care about that, so this "we fixed insecurities" clause doesn't tell them anything.

We could either say "we found a way to hide your data from other posthog users when making sql queries", but that sounds like table stakes. So instead I'd just remove this bit... 🤔